project-fob/Controllers/StatsController.cs

```diff
@@ -23,39 +23,36 @@ public ActionResult Index()
             return View();
         }
         public string GetStats()
-        {
-            { //- for in the same topic, : for different topic, ; for completely different statistic
-
-                byte[] meetingIdValue;
-                bool gotvalue = false;
-                gotvalue = HttpContext.Session.TryGetValue("meetingid", out meetingIdValue);
+        { //- for in the same topic, : for different topic, ; for completely different statistic
 
-                string session = meetingIdValue.ToString();
-                //string session = Session["meetingid"].ToString();
+            bool gotvalue = false;
+            gotvalue = HttpContext.Session.TryGetValue("meetingid", out var meetingIdValue);
 
-                string MeetingIdString = System.Text.Encoding.ASCII.GetString(meetingIdValue);
-                
-                Meeting meeting = db.Meeting.Include(x => x.Stats).Single(m => m.MeetingId == MeetingIdString);
+            string session = meetingIdValue.ToString();
+            //string session = Session["meetingid"].ToString();
```

Comment:
In most cases, it's best to just deleted code rather than comment it out. Otherwise the codebase gets messy very quickly, it's not quite [technical debt](https://en.wikipedia.org/wiki/Technical_debt), but it does still get in the way when you're reading the code.

Reply:
Didn't knew this term actually exist.

Reply:
In that case, it's good to find out about it now :) It's a big reason why large software projects end up in such a mess, and why it takes longer and longer to continue working on projects that have been around for a while. You need to be quite aggressive to stay on top of technical debt and keep things maintabile.

It's also closely related to [refactoring](https://en.wikipedia.org/wiki/Code_refactoring)